source/extensions/filters/http/cache/cache_policy.h

```diff
@@ -0,0 +1,98 @@
+#pragma once
+
+#include "envoy/http/header_map.h"
+#include "envoy/stream_info/filter_state.h"
+
+#include "source/extensions/filters/http/cache/cache_headers_utils.h"
+#include "source/extensions/filters/http/cache/http_cache.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Cache {
+
+struct CacheEntryUsability {
```

Comment:
Since this is going to be returned by potential CachePolicy extensions, could you add some comments on the status and age field, to help understand what they are and how they are supposed to be used?